recipes/ut/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import os
+import glob
+
+
+class UTConan(ConanFile):
+    name = "ut"
```

Comment:
I wonder if this name is too generic 🤔 

perhaps `boost-ext-ut` so it's easy for consumers to find?

Reply:
I thought that might be an issue. I think that's a good idea.